src/NovelRT/Graphics/Camera.cpp

```diff
@@ -15,6 +15,13 @@ namespace NovelRT::Graphics {
     return returnVal;
   }
 
+  std::unique_ptr<Camera> Camera::createDefaultPerspectiveProjection(const Maths::GeoVector2<float>& windowSize) {
+    auto returnVal = std::make_unique<Camera>();
+    returnVal->setProjectionMatrix(Maths::GeoMatrix4x4<float>(glm::perspective<float>(glm::radians(90.0f), windowSize.getX() / windowSize.getY(), 0.1f, 100.0f)));
+    returnVal->setViewMatrix(Maths::GeoMatrix4x4<float>(glm::lookAt(glm::vec3(0.0f, 0.0f, 3.0f), glm::vec3(0.0f, 0.0f, 0.0f), glm::vec3(0.0f, 1.0f, 0.0f))));
```

Comment:
Do you want me to comment the role of each vec3?

Reply:
What do you mean?

Reply:
In case what they are for is unclear?

Reply:
ok maybe im being dumb - are you talking about commenting API surface or implementation details?

Reply:
Implementation details, for other contributors, don't know if that's a thing

Reply:
In this case, GLM is its own documented API. I know what that does, the developers know what that does, and I'd expect anyone wanting to touch the graphics internals to know what it does or to at least go read up about it.

_However._

It might not hurt to add any missing Doxygen documentation to the _public_ API surface while you are here to the camera, which describes what the API surface does. :D